externalcommands/manpages/man/page-format

```diff
@@ -0,0 +1,28 @@
+Pages in @hman@n are text files encoded in ISO-8859-15 encoding (ASCII-only UTF-8 is also acceptable). Files have to be either @pLF@n or @pCR@n - @pCRLF@n will generate @etwo@n line breaks!
+
+In addition to normal text, page files also contain so-called @hFormater Flags@n. They are sequences of characters, that start with @p@@@n symbol.
+
+Here's a list of all formater flags:
+
+@h@@N@n   - Sets the color to normal text color
```

Comment:
I kinda like the choice here for selecting the colors, better than what I added to echo recently.
However, should we be thinking about using the same escape character perhaps?  Echo uses the backslash which is a common choice for escape character.
I'm curious as to why tho chose the @

Reply:
> I'm curious as to why tho chose the @

idk, i thought it's useless enough. 

> use backslash

Idk, I feel like I need to obey POSIX standards when I end up using backslash (actually, idk, if POSIX is the standard that regulates the backslash thing or some different standard regulates them, but anyways). I mean, if i implement \\xHH, does this mean, i also need to implement \\n? can i use \\b or or should i avoid it due to the standard?

Also, it's called **escape** character for a reason - the primary use of \\ is to prevent a special character from doing its usual thing. in my case however, there are no special characters outside of `@`. i guess if the lone `;` start a comment, then it'd make sense to add `\;` as a way to stop `;` from commenting.